java-codegen/src/main/java/org/opensearch/client/codegen/utils/Versions.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.client.codegen.utils;
+
+import javax.annotation.Nonnull;
+import org.semver4j.Semver;
+
+public final class Versions {
+    private Versions() {}
+
+    public static final Semver V2_0_0 = Semver.of(2, 0, 0);
```

Comment:
@Xtansia so we effectively 3 major versions now: 1.x , 2.x  - released, 3.x - unreleased. Why we only consider 2.0.0 here?

Reply:
@reta This was primarily introduced for handling the fact `WriteResponseBase` used to include the doc `type` field  which has been already been removed in the client. Mainly trying to keep things as back-portable as possible for `2.x` for now. Can always expand upon this as the need arises